src/style.css

```diff
@@ -0,0 +1,58 @@
+html,
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+}
+
+.stars:not(:first-child) {
+  margin-top: -6px;
+}
+
+.stars--star {
+  width: 20px;
+  height: 20px;
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  margin-left: 2px;
+  margin-top: 2px;
+}
+
+.stars--1 > .stars--star:first-child {
+  width: 20px;
+  height: 20px;
+  background-image: url("images/star-active.svg");
+  background-repeat: no-repeat;
+}
+
+.stars--2 > .stars--star:nth-child(-n + 2) {
+  width: 20px;
+  height: 20px;
+  background-image: url("images/star-active.svg");
+  background-repeat: no-repeat;
+}
+
+.stars--3 > .stars--star:nth-child(-n + 3) {
+  width: 20px;
+  height: 20px;
+  background-image: url("images/star-active.svg");
+  background-repeat: no-repeat;
+}
+
+.stars--4 > .stars--star:nth-child(-n + 4) {
+  width: 20px;
+  height: 20px;
+  background-image: url("images/star-active.svg");
+  background-repeat: no-repeat;
+}
+
+.stars--5 > .stars--star:nth-child(-n + 5) {
```

Comment:
It is a good example when you actually can group selector because they MUST have the same styling and one can't change without the others